identity/app/services/DiscusssionApiService.scala

```diff
@@ -0,0 +1,53 @@
+package services
+
+import play.api.libs.json.{Format, JsValue, Json}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+import play.api.libs.ws.{WSClient, WSResponse}
+import utils.SafeLogging
+import cats.implicits._
+import cats.data.EitherT
+
+// A user will exist in Discussion if they have commented or recommended a comment.
+// DAPI does not expose recommendation stats by user, so we only rely on comments to decide whether or not a user in Discussion should have a public profile
```

Comment:
Could we add a point that says `users need 1 or more comments to have a public profile page` so its explicit what the rule is?